pyof/utils.py

```diff
@@ -0,0 +1,37 @@
+"""General Unpack utils for python-openflow."""
```

Comment:
this is already clear in the corresponding issues mentioned here:
https://github.com/kytos/python-openflow/issues/396 and https://github.com/kytos/python-openflow/issues/381